src/main.ts

```diff
@@ -174,8 +142,15 @@ export async function run(): Promise<void> {
     }
 
     // Set url as output.
-    // TODO: update this to use JSON or YAML machine-readable output instead.
-    setUrlOutput(output.stdout + output.stderr);
+    const response = parseDeployResponse(output.stdout);
+    if (response) {
+      setOutput('name', response.name);
+      setOutput('serviceAccountEmail', response.serviceAccountEmail);
+      setOutput('versionURL', response.versionURL);
+      setOutput('url', response.versionURL);
```

Comment:
Will these ever be different, say when promote is false?

Reply:
Yea - they are not provided with `--no-promote`. I have a warning message.

Reply:
I meant `https://....appspot.com/v1/app` vs `https://....appspot.com/app` would we care about the difference? Seems like it might be useful, IE main url vs version url (im not even sure if this is given in output).

Reply:
Oh sorry. They are both the same. `url` is there for backwards compat, but it's always been the version url.